src/app/views/experiment_page.js

```diff
@@ -0,0 +1,68 @@
+import View from 'ampersand-view';
+import dom from 'ampersand-dom';
+import mustache from 'mustache';
+import webChannel from '../lib/web_channel';
+
+export default View.extend({
+  template: mustache.render(`<section class="page" data-hook="experiment-page">
+               <h1 data-hook="displayName"></h1>
+               <button data-hook="install">Install</button>
+             </section>`),
+
+  events: {
+    'click button': 'onClick'
+  },
+
+  initialize(opts) {
+    View.prototype.initialize.apply(this, arguments);
+    if (opts && opts.experiment) {
+      this.model = opts.experiment;
+    }
+  },
+
+  render(opts) {
+    View.prototype.render.apply(this, arguments);
+    this.renderWithTemplate(this);
+
+    if (opts && opts.experiment) {
+      this.model = opts.experiment;
+    }
+    this.heading = this.el.querySelector('[data-hook=displayName]');
+    dom.text(this.heading, this.model.displayName);
+
+    this.button = this.el.querySelector('button');
+    this.renderButton();
+  },
+
+  // TODO: move into a base view?
+  remove() {
+    const parent = this.el.parentNode;
+    if (parent) parent.removeChild(this.el);
+  },
+
+  renderButton() {
+    if (this.model.isInstalled) {
+      dom.setAttribute(this.button, 'data-hook', 'uninstall');
+      dom.text(this.button, 'Uninstall');
+    } else {
+      dom.setAttribute(this.button, 'data-hook', 'install');
+      dom.text(this.button, 'Install');
+    }
+  },
+
+  onClick(evt) {
```

Comment:
I prefer for these to be named after the action being performed rather than named after the event: `install` vs `onClick`. I think it makes the code more self documenting and helps with name collision if you have multiple click handlers.